extending.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
```

Comment:
If this is going to go in a separate, informal, document, then:

1. It should go in /docs.
2. I think it should just be a plain Markdown file (like the other informal docs in that directory), rather than a Respec-formatted doc.